Basic/APIDiorama/Assets/Common/Scripts/com.unity.samples.api-diorama.asmdef

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "com.unity.samples.api-diorama",
```

Comment:
nitpick: I'd call this "common"? Since the other assemblies also have that name as a prefix?

Reply:
*add common at the end I mean